submissions/tavor118sn/task_js_dom/script.js

```diff
@@ -0,0 +1,57 @@
+const superheroes = [
+  {
+    name: 'batman',
+    logo: './img/batman-logo.png',
+    img: './img/batman-img.png',
+    description: 'Batman has been Gotham City\'s protector for decades, CEO of Wayne Enterprises, Patriarch of the Bat Family and veteran member of the Justice League.',
+  },
+  {
+    name: 'superman',
+    logo: './img/superman-logo.png',
+    img: './img/superman-img.png',
+    description: 'Superman is an alien named Kal-El from the destroyed planet Krypton. As an adult, Superman became the protector of Earth, working at the Daily Planet as Clark Kent.',
+  },
+];
+
+const menuButtonTemplate = (name) => {
+  return `
+  <li class ="nav-item">
+     <button id="${name}-btn" class="nav-btn">${name}</button>
+  </li>`;
+};
+
+function createMenu(superheroes) {
+  let menuItems = superheroes.map(
+    ({name}) => menuButtonTemplate(name)
+  );
+  let navMenu = document.querySelector(".nav-list");
+  navMenu.innerHTML = menuItems.join('');
+}
+
+const fillContent = (item) => {
+  let logo = document.querySelector(".logo");
```

Comment:
This code will run on every menu click, but reference to logo element will stay the same. DOM operations are expensive and it is not necessary to query DOM for this element on every click.

Reply:
Created `consts` for those elements.